### PR TITLE
WL-4965 Add recorded lectures tool and make available in RHS user menu.

### PR DESCRIPTION
**Pull request by RebeccaMiller-Which**

### CODE REVIEW

File: portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/MoreSiteViewImpl.java [hunk 1]
```diff
@@ -154,6 +157,9 @@ public Object getRenderContextObject()
                             } else if ( preferencesToolId.equals(placement.getToolId()) ) {
                                 prefsToolUrl = Web.returnUrl(request, "/site/" + Web.escapeUrl(siteHelper.getSiteEffectiveId(s)) + "/page/" + Web.escapeUrl(p.getId()));
                                 mrphs_prefsToolUrl = Web.returnUrl(request, "/site/" + Web.escapeUrl(siteHelper.getSiteEffectiveId(s)) + "/tool-reset/" + Web.escapeUrl(placement.getId()));
+							} else if ( recordedLecturesToolId.equals(placement.getToolId()) ) {
```
Comment (buckett) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/MoreSiteViewImpl.java]: Indentation looks wrong.

File: web/web-portlet/src/webapp/WEB-INF/sakai/PortletIFrame.xml [hunk 2]
```diff
@@ -95,4 +95,22 @@
...
+			title="Recorded Lectures"
+			description="For accessing all recorded lectures relevant to the student.">
+		<category name="myworkspace" />
+		<configuration name="source" value="/access/content/public/recorded-lectures.html" type="final"/>
+		<configuration name="height" value="600px" />
+
+		<configuration name="reset.button" value="false" type="final" />
+
+		<!-- Event for accessing the sakai.iframe.annotatedurl tool; remove property to remove logging of event -->
+		<configuration name="webcontent.read" value="webcontent.myworkspace.read" />
```
Comment (buckett) [inline web/web-portlet/src/webapp/WEB-INF/sakai/PortletIFrame.xml]: I think these events should be changed to something more specific to indicate it's the recorded lectures tool.

**Comment by buckett** [comment]
@RebeccaOxfordUni Still waiting on comments.

**Comment by RebeccaMiller-Which** [comment]
I've made the changes you suggested Matthew.

File: web/web-portlet/src/webapp/WEB-INF/sakai/PortletIFrame.xml [hunk 3]
```diff
@@ -95,4 +95,22 @@
...
+		<configuration name="source" value="/access/content/public/recorded-lectures.html" type="final"/>
+		<configuration name="height" value="600px" />
+
+		<configuration name="reset.button" value="false" type="final" />
+
+		<!-- Event for accessing the sakai.iframe.annotatedurl tool; remove property to remove logging of event -->
+		<configuration name="webcontent.read" value="webcontent.recorded.lectures.read" />
+
+		<!-- Event for modifying the sakai.iframe.annotatedurl tool configuration; remove property to remove logging of event -->
+		<configuration name="webcontent.revise" value="webcontent.recorded.lectures.revise" />
```
Comment (buckett) [inline web/web-portlet/src/webapp/WEB-INF/sakai/PortletIFrame.xml]: I think this needs to be 32 characters or less.

**Comment by RebeccaMiller-Which** [comment]
Event name changed.

**Comment by buckett** [comment]
Magic, looks good.

**Pull request merged**